src/components/Navigation/NavigationItemTrigger/_navigationItemTrigger.scss

```diff
@@ -45,6 +45,7 @@
       transition:
         max-height 0.3s cubic-bezier(0.4, 0, 0.2, 1),
         opacity 0.2s cubic-bezier(0.65, 0, 0.35, 1);
+      transition-delay: 0.2s;
```

Comment:
We may want to increase the delay and remove the delay when the parent LI is hovered.